.build/deps.mk

```diff
@@ -18,6 +18,7 @@ deps: libdeps
 	$(ECHO_V)go install ./vendor/github.com/sectioneight/md-to-godoc
 	@$(call label,Installing interfacer...)
 	$(ECHO_V)go install ./vendor/github.com/mvdan/interfacer/cmd/interfacer
+	$(ECHO_V)richgo testfilter 2>/dev/null || ($(call label,Installing richgo) && go get github.com/sectioneight/richgo)
```

Comment:
Note: intentionally not vendoring this and just testing for the `testfilter` command because I don't want to encourage living on my fork for any longer than required